app/build.gradle

```diff
@@ -128,6 +128,8 @@ dependencies {
 
     // (Java only)
     implementation "androidx.work:work-runtime:$work_version"
+
```

Comment:
@bananateg : Please don't use third party libraries, they can cause issues later or not. Unless we can't achieve the same through native , please refrain from using them.

Reply:
I agree with @aakash1313 let's find a way to use native libraries.
Using a third party library for our calendar has caused many crashes in production.

Reply:
ok